src/styles/linkWithArrow.module.scss

```diff
@@ -0,0 +1,20 @@
+.link-with-arrow {
+  color: var.$color-primary--dark;
+  font-size: var.$fs-base;
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+
+  &--bigger {
+    @extend .link-with-arrow;
+    font-size: var.$fs-body;
+  }
+
+  &:focus {
+    text-decoration: underline;
+  }
+}
+
+.link-text {
+  margin-right: 10px;
```

Comment:
This should be changed to a variable (so something like var.$space or even making it a new variable to reference that), also I like to use rem-units instead of px units for better responsiveness and accessibility